_transcripts/015-twir-338.md

```diff
@@ -0,0 +1,56 @@
+
+If you have always wanted to contribute to Open Source projects, but don’t know where to start, look in the Call for Participation section for tasks that you can pick up and get started with. This week we feature an issue from the clap project - the command line argument parser for Rust - which has a mentor available to help. We also have a couple of issues from the keikan project - an elegant rendering engine written in Rust. Both of these projects are great opportunities to get involved with Open Source software and the Rust language.
+
+The RFC - or request for comment process - is at the core of how the Rust project operates. This week, the RFC which proposes transitioning to Rust-analyzer as our official Language Server Protocol (or LSP) implementation is in final comment period, which means it is reaching a decision soon. If you have an opinion on this, now is the time to express it in that RFC. The link to it is also in the show notes. 
+
+There is also a new RFC about reading into uninitialized buffers. This one focuses on the Read trait - specifically how when you use it it requires that the buffer passed to its various methods be pre-initialized even though the contents will be immediately overwritten. This RFC proposes an interface to allow implementors and consumers of Read types to robustly and soundly work with uninitialized buffers. If you are curious about this, if you have opinions, thoughts, or questions, definitely head to that RFC and comment.
+
+Finally, there are several upcoming events in the Rust community. This week and next there are a lot of online meetups from across the world that are open to all. This includes meetups in Vancouver BC, Turin IT, Dallas TX, Berlin, and Montreal. There is also a meetup in Durham, NC, called the Triangle Rustaceans. Meetups are a great way to not only learn Rust, but also connect with the fantastic community that has made Rust so successful. And because many of these meetups are now online, you can connect with Rust communities from around the world.
+
+That’s all for this week’s podcast - there are a lot more items in this week’s newsletter, make sure to read it to keep up with all the awesome things going on in the Rust world.
+
+This Week in Rust is edited by myself, Andre Bogus, and Colton Donnelly. This week’s newsletter’s contributors include, using their GitHub usernames, mrcosta, nickwilcox, pksunkar, and zoni! Thank you so much to everyone who contributes to This Week in Rust!
+
+Thank you also to the Rustacean station for hosting this podcast! I will be back next week bringing you more of the wonderful news from the wonder world of Rust. Take care everyone.
+
+Show Notes
```

Comment:
The show notes down here can probably be removed, since they are already in the main show notes :)

Reply:
And if any of it isn't, then probably best to fold them into the show notes. The transcript should ideally be _just_ transcript.

Reply:
got it!